packages/xo-server/src/api/sr.js

```diff
@@ -719,7 +719,7 @@ export async function probeIscsiLuns({
       id: lun.LUNid.trim(),
       vendor: lun.vendor.trim(),
       serial: lun.serial.trim(),
-      size: lun.size.trim(),
+      size: lun.size?.trim(),
```

Comment:
Can `serial` or  `vendor` be `undefined`?

Reply:
We are not sure about nothing. 

This PR isn't a fix but a work-around for an edge case. Generally, when we cannot get the lun size, the SR will not be created due to unavailability reason.